### ISSUE TITLE
Remix Link component + Storybook (Webpack) builder error.

### ISSUE THREAD

**Issue by romanpeganov** [remix-run/remix#4584]
### What version of Remix are you using?

1.7.5

### Steps to Reproduce

Hello,
First of all, let me thank you for such a wonderful tool as Remix.

I use Storybook to create my React components. In the current project, I create an application with Remix. In one of the my components, I used the Link component from "@remix-run/react". The Storybook builder could not compile such a component by issuing an error:

"ERROR in ./node_modules/@remix-run/react/dist/esm/components.js 752:45
Module parse failed: Unexpected token (752:45)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders"

In the specified file, I found a code that caused an error: 
`let resolvedPath = useResolvedPath(action ?? ".");`
I corrected this code a little and the error has disappeared: 
`let resolvedPath = useResolvedPath(action ? action : ".");`

That's all!

PS: Node: 16.1.3, Npm: 8.1.2, Remix: 1.7.5, Storybook: 6.5.13 (with Webpack 5.0). Storybook in this version can work with this environment only with npm flag 'legacy-peer-deps=true'.

PSS: For those who are faced with errors of the React router into the 'stories': for the correct use of the 'Link' components with the Storybook, wrap the components in the 'stories' files, in the 'BrowserRouter' component: 
```
import { BrowserRouter } from "react-router-dom"; 
const Template = () => (
  <BrowserRouter>
    <ComponentWithLinks />
  </BrowserRouter>
);
```

### Expected Behavior

./node_modules/@remix-run/react/dist/esm/components.js 752:45
`let resolvedPath = useResolvedPath(action ? action : ".");`

### Actual Behavior

./node_modules/@remix-run/react/dist/esm/components.js 752:45
let resolvedPath = useResolvedPath(action ?? ".");

**Comment by machour** [comment]
Hi @romanpeganov and thank you for the detailed report and resolution path. We merged a fix (https://github.com/remix-run/remix/pull/4561) for the problem you've spotted and it should be released in 1.7.6 soon! Leaving this open until then.

**Comment by machour** [comment]
1.7.6 have been published with the fix.
Please open a new issue if you're still seeing a problem 🙏🏼
